Sources/Caching/DeviceCache.swift

```diff
@@ -142,25 +139,48 @@ class DeviceCache {
         }
     }
 
-    // MARK: - offerings
+    // MARK: - Offerings
 
-    func cache(offerings: Offerings) {
-        offeringsCachedObject.cache(instance: offerings)
+    func cachedOfferingsData(appUserID: String) -> Data? {
```

Comment:
I think these function names can be confusing... As in, there is nothing there that tells us if it's using the memory or the disk cache and you need to look at the implementation.

Reply:
Good call, will improve.